modules/ln_glgui/glgui.scm

```diff
@@ -264,8 +303,11 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
                   (apply glgui:inputloop (append (list t x0 y0) gs)))))
           (if (fx= t EVENT_REDRAW)
               (wait-for-time-or-signal!)
-              (reset-wait!))))
+              (begin
+                (thread-sleep! step)
```

Comment:
If you set frame-period-custom to a custom function, this doesn't update step so you have different timing behaviours when running in `CONSOLEAPP`s or not?